### PR TITLE
Refactor and Expose PageHeader for plugins

### PR DESCRIPTION
**Pull request by daiyiwhale**
- light refactor of `PageHeader` using newly uncovered full-blown `classnames` features 
- remove specific style `icon-app-container`
- expose `PageHeader` for plugin utilisation

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @kennyt

File: src/js/plugin-bridge/PluginModules.js [hunk 1]
```diff
@@ -36,26 +36,27 @@ module.exports = {
     TabsMixin: 'TabsMixin'
   },
   components: {
+    ActionsModal: 'modals/ActionsModal',
```
Comment (kennyt) [inline src/js/plugin-bridge/PluginModules.js]: this is all :abc: + adding pageheader right?

Comment (daiyiwhale) [inline src/js/plugin-bridge/PluginModules.js]: yep!

**Comment by jfurrow** [comment]
LGTM! 🚀

File: src/js/components/PageHeader.js [hunk 2]
```diff
@@ -37,34 +52,51 @@ class PageHeader extends React.Component {
...
+      'container-pod container-pod-short flush-top',
+      'container-pod-divider-bottom container-pod-divider-bottom-align-right',
+      'container-pod-divider-inverse',
+      dividerClassName
+    );
+
+    let mediaWrapperClasses = classNames(
+      'media-object-spacing-wrapper media-object-spacing-narrow',
+      'media-object-offset',
+      mediaWrapperClassName
```
Comment (kennyt) [inline src/js/components/PageHeader.js]: all of these are able to be objects now, in order to be able to remove the previous classes. you'd also need to change the `PropTypes`

Comment (daiyiwhale) [inline src/js/components/PageHeader.js]: good catch!

Comment (daiyiwhale) [inline src/js/components/PageHeader.js]: fix'd

Comment (kennyt) [inline src/js/components/PageHeader.js]: please change the propTypes to do PropTypes.oneOfType etc..

Comment (daiyiwhale) [inline src/js/components/PageHeader.js]: I did that, I think?

Comment (kennyt) [inline src/js/components/PageHeader.js]: kk cool, didn't see it before

**Comment by kennyt** [comment]
if tests pass i'd merge

**Comment by daiyiwhale** [comment]
What the


Build restarted!

**Pull request merged**